lib/cli.js

```diff
@@ -60,8 +60,8 @@ function quietFixPredicate(message) {
 /**
  * Translates the CLI options into the options expected by the ESLint constructor.
  * @param {ParsedCLIOptions} cliOptions The CLI options to translate.
- * @param {"flat"|"eslintrc"} [configType="eslintrc"] The format of the
```

Comment:
I think this is happening because the parameter doesn't specify a default in the code. Can we add that in the code so the JSDoc is accurate?